utils/ga.js

```diff
@@ -0,0 +1,17 @@
+export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS
```

Comment:
@jbusecke, I'm currently using a Google Analytics ID that I set up under my account. Once you are back, we can easily replace it with one under LEAP or your account.